docs/model/rules-to-model.md

```diff
@@ -0,0 +1,55 @@
+# Rules to a model
```

Comment:
I'm missing here the rule that Model can depend on a Component, but not vice versa.